docs/tutorial.rst

```diff
@@ -359,6 +359,13 @@ fact edible).
 effect that is present (e.g. believing a mushroom is edible when it is
 in fact poisonous).
 
+Quick Note for Jupyter Notebook users:
```

Comment:
@chrisfs Great catch! Other sprinters ran into this issue also.  

To fix this, we suggest that you use  RST NOTE formatting to include this information.  It looks like this:
`.. NOTE:: Jupyter, Pandas, and other ancillary....`

An example of this formatting can be found in the quickstart docs.  The note is located right above the installation section.  Here is a link to the raw text and a link to the displayed text.  

displayed - http://www.scikit-yb.org/en/latest/quickstart.html
raw text - https://raw.githubusercontent.com/DistrictDataLabs/yellowbrick/develop/docs/quickstart.rst